client/src/modules/stock/entry/modals/StockEntryModalForm.service.js

```diff
@@ -77,7 +77,7 @@ function StockEntryModalForm(uuid) {
     row._error = null;
 
     // if the stock does not expire/have an expiration date, generate a fake one
-    if (!this.opts.expires) {
+    if (!this.opts.tracking_expiration) {
```

Comment:
This change breaks the expiration dates completely.  Can you please double-check this?  I reported it last review, but it wasn't changed.

Reply:
okay